### ISSUE TITLE
Consider changing the default value of `awaitPromise` to false

### ISSUE THREAD

**Issue by juliandescottes** [w3c/webdriver-bidi#238]
The `awaitPromise` parameter used in [script.evaluate](https://w3c.github.io/webdriver-bidi/#command-script-evaluate) and [script.callFunction](https://w3c.github.io/webdriver-bidi/#command-script-callFunction) is a boolean which defaults to true. 

It sounds like a sensible default behavior, but optional boolean parameters which default to true can be confusing. Developers commonly assume that if a flag is not provided, it will be falsy.

Could we make it default to false? Or make it mandatory? 

cc @sadym-chromium

**Comment by jgraham** [comment]
I tend to agree that we should make it a general rule that missing booleans default to false (and there are probably other similar rules for simple types), and should update the existing spec to match these rules.

**Comment by sadym-chromium** [comment]
The default behaviour should hide unnecessary complexity from the users don't need them. I'd assume `awaitPromise=false` is more extravagant use-case then `awaitPromise=true`, so keeping the behaviour `awaitPromise=true` by default makes more sense for my mind. Flipping the name to something like `donNotWaitForPromise` seems quite confusing as well.

**Comment by juliandescottes** [comment]
Thanks for the feedback! I agree switching the default behavior is not great. Making it mandatory would partially address that but it's making the API more verbose for the sake of clarity. We could find a shorter name, `noAwait`, `sync`, ...?

If we are saying that for this parameter the default behavior is really what most of the consumers should be happy with, then making the parameter name slightly longer / obscure is not necessarily a bad thing?

**Comment by sadym-chromium** [comment]
Solved by making the field obligated
